### ISSUE TITLE
Log label level is not filtered by FiltrationMode

### ISSUE THREAD

**Issue by Mario-Hofstaetter** [mishamyte/serilog-sinks-grafana-loki#9]
A small detail I noticed while at #7:

Version `4.0.1`. When using 

```csharp
.WriteTo.GrafanaLoki("http://localhost:3100",
	filtrationLabels: new string[] {},
	filtrationMode: LokiLabelFiltrationMode.Include,
	outputTemplate:LOKI_LOG_TEMPLATE)
```

the `level` still shows up as `Label` in loki. Is this intentional?
This the log level considered *special* and exempt from the filtration?

![grafik](https://user-images.githubusercontent.com/33002073/99160304-347ef080-26e6-11eb-8ca7-6dc6d7943b43.png)

Also, were is the lower-case "`level`" and "`error`" format coming from?
The default output of `{Level}` for Console and File (`JsonFormatter`) sink seems to be "`Level`" and "`Error`".

Loki seems to be case-sensitive, so neither `{level="Error"}` nor `{Level="error"}` is working in that case.

**Comment by mishamyte** [comment]
Since first of all it is a sink for logger, it is reasonable to keep a log level.
It is designed, but I forgot to write it while implementing #4 . I think, I need to write a documentation for filtration feature.

`level` label comes out from Grafana [log level](https://grafana.com/docs/grafana/latest/panels/visualizations/logs-panel/#log-level) feature. It is used by Grafana for visualization via color. Also a value (`error`) comes out from Grafana API - [LogLevel enum](https://grafana.com/docs/grafana/latest/packages_api/data/loglevel/)

Mapping rules described in this helper - [https://github.com/mishamyte/serilog-sinks-grafana-loki/blob/master/src/Serilog.Sinks.Grafana.Loki/Utils/LogEventLevelExtensions.cs](https://github.com/mishamyte/serilog-sinks-grafana-loki/blob/master/src/Serilog.Sinks.Grafana.Loki/Utils/LogEventLevelExtensions.cs)

**Comment by Mario-Hofstaetter** [comment]
Thanks for the detailed answer and insight.

Is there a possibilty to force lower case label key name for all properties?
What would you recommend @mishamyte ? Do you have loki and grafana deployed to be used by others?
It could be confusing long term for users that properties like `Resource`,`SourceContext`, ... are PascalCase,
and only `level` is all lowercase.

**Comment by mishamyte** [comment]
1. We've used Loki + Grafana on go-live project (about 15 .NET microservices + 2 Node.Js) for monitoring & logging all service info + infra. Later we switched to Seq for logs (we faced some troubles with high cardinality on non-prod and had no expertise plus time for investigation (it was a deadlines time). So the fast-rescue decision was taken) and Loki remained for infra. While using all was OK, cause the most of team is .NET devs and PascalCase for props is native for 'em)
2. As I know a common approach for labels names is `kebab-case`
3. We could do this enhancement, but it could be a lil' bit tricky. Cause if we would have this feature (for example let's call it LabelCase). If user would use it with a filtration feature. In this case - how filtration list should be filled? In logger prop naming style - PascalCase or in selected case? And what about global labels?

So, the example pseudocode:
`.WriteTo.GrafanaLoki("http://localhost:3100",
	filtrationLabels: new string[] { **"ShouldIBeInPascal", "should-i-be-in-kebab"** },
	filtrationMode: LokiLabelFiltrationMode.Include,
        labelCase: Kebab)`
